### ISSUE TITLE
python3.5 warning

### ISSUE THREAD

**Issue by Evidlo** [h5py/h5py#1026]
* h5py 2.7.1
* Fedora 25
* Python 3.5

In python3.5, `import h5py` yields the following warning:

```
/home/evan/resources/venv3/lib/python3.5/site-packages/h5py/__init__.py:36: 
FutureWarning: Conversion of the second argument of issubdtype from `float` 
to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
```

**Comment by nicain** [comment]
Can anyone suggest if this is an important warning, and if so, how to remediate?

**Comment by aragilar** [comment]
Duplicate of #961, fixed in the 2.8.0 rc. You can ignore the error.
